README.md

```diff
@@ -270,6 +270,8 @@ _\*_ v3 support is limited to the subset of functionality required to enable lan
 | Linux   | arm64        | ✅         | 9.x +            |
 | Windows | x86_64       | ✅         | All              |
 | Windows | x86          | ✅         | 9.x -            |
+| Alpine  | x86_64       | ✅         | 10.x +           |
+| Alpine  | arm64        | ✅         | 10.x +           |
```

Comment:
I'm not sure about 9.x and below. I think Ruby standalone did support it, right?

Reply:
Yes, it was supported with a workaround

for 9.x and below, users should be noted that they require extra dependencies, as the pact-ruby-standalone package is glibc based

https://github.com/pact-foundation/pact-ruby-standalone/wiki/Using-the-pact-ruby-standalone-with-Alpine-Linux-Docker

Reply:
perfect thanks @tienvx